src/c/inteldeflater/IntelDeflater.c

```diff
@@ -90,10 +174,11 @@ Java_htsjdk_samtools_util_zip_IntelDeflater_init(JNIEnv *env, jclass cls, jint l
             return jlong_zero;
         }
     }
+  }
 }
 
 JNIEXPORT void JNICALL
-Java_htsjdk_samtools_util_zip_IntelDeflater_setDictionary(JNIEnv *env, jclass cls, jlong addr,
+Java_net_sf_samtools_util_zip_IntelDeflater_setDictionary(JNIEnv *env, jclass cls, jlong addr,
```

Comment:
Why are these all renamed to `net_sf_samtools` from `htsjdk_samtools`?  The package name in the java code is `htsjdk.samtools`.